examples/bridge-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -23,6 +23,7 @@
 #include <app/common/gen/cluster-id.h>
 #include <app/server/Mdns.h>
 #include <support/CodeUtils.h>
+#include "Bluetooth.h"
```

Comment:
Shouldn't this be <Bluetooth.h> like above?

Reply:
Done